src/core/http_loader_test.cpp

```diff
@@ -12,9 +12,9 @@ using namespace mavsdk;
 
 class HttpLoaderTest : public testing::Test {
 protected:
-    const std::string _file_url_1 = "http://subdomain.domain.com/some_folder/some_file";
-    const std::string _file_url_2 = "http://subdomain.domain.com/some_folder/another_file.txt";
-    const std::string _file_url_3 = "http://subdomain.domain.com/some_folder/yet_another_file.mp4";
+    const std::string _file_url_1 = "https://subdomain.domain.com/some_folder/some_file";
```

Comment:
Does it actually make a difference? If yes, shouldn't we both test for `http` and `https`?

Reply:
Hehe good point. That change is probably a bit over the top.